MMCore/MMEventCallback.h

```diff
@@ -89,4 +89,18 @@ class MMEventCallback
       std::cout << "onSLMExposureChanged()" << name << " " << newExposure << "\n";
    }
 
+   virtual void onDeviceLoaded(const char* name)
+   {
+      std::cout << "onDeviceLoaded " << name;
+   }
+
+   virtual void onDeviceWillBeUnloaded(const char* name)
+   {
+      std::cout << "onDeviceWillBeUnloaded " << name;
+   }
+
+   virtual void onAllDevicesWillBeUnloaded()
+   {
+      std::cout << "onAllDevicesWillBeUnloaded ";
+   }

```

Comment:
Ideally I think this would be made irrelevant by the single device unloading signal. But that doesn't look like it's possible without pushing the external callback into the deviceManager which currently doesn't happen. Happy to do that, but figured Id start with the less invasive option.